pydantic_ai_slim/pydantic_ai/models/mistral.py

```diff
@@ -227,13 +256,25 @@ async def _completions_create(
         if model_request_parameters.builtin_tools:
             raise UserError('Mistral does not support built-in tools')
 
+        # Determine the response format based on output mode
+        response_format = self._get_response_format(model_request_parameters)
+
+        # When using native JSON schema mode, don't use tool-based output
```

Comment:
I don't think we need to branch on this: if `model_request_parameters.output_mode == 'native'`, `model_request_parameters.output_tools` will be empty anyway. Note that we don't have logic like this in the other models